ci-operator/step-registry/single-node/conf/e2e/graceful-shutdown/single-node-conf-e2e-graceful-shutdown-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
```

Comment:
This step is unused, even though it should be the one that is used because the aws-one is no longer really aws specifically, and thus should be removed

Reply:
(At least that's how I understand it, please correct me if I'm wrong)